components/list/list-item-generic-layout.js

```diff
@@ -66,6 +77,9 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 				grid-column: actions-start / actions-end;
 				z-index: 4;
 			}
+			.focused::slotted([slot="actions"]) {
```

Comment:
Same thing for this one - there is a rule that enforces class names to begin with `d2l-`.  It doesn't protect us 100%, but it's better than nothing.  The rationale is - for legacy Edge, which does not support shadowDOM, elements within components are possible targets for such selectors in the page.

Reply:
This is now using an attribute on the host to target